app/wama.web/Controllers/CheckInController.cs

```diff
@@ -44,6 +47,14 @@ public IActionResult Index()
                 {
                     SetErrorMessages(AppString.AccountPendingApprovalMessage);
                 }
+                else if (userAccount.IsSuspended)
+                {
+                    SetErrorMessages(AppString.AccountSuspended);
+                }
+                else if (certificateInfo != null && System.DateTimeOffset.Compare(System.DateTimeOffset.Now, certificateInfo.ExpiresOn) > 0)
```

Comment:
The name certificationInfo can be simplified to certification. It's less redundant.